app/models/patron.rb

```diff
@@ -259,9 +246,7 @@ def can_schedule_pickup?(library)
   end
 
   def can_schedule_special_collections_visit?
```

Comment:
I kept this as-is for the time being (trying to keep the changes to a minimum-ish), but imagine it could get rolled up into one of the methods above.